Bottom: 29fdbfc2381ca6238bb2fcee970c7ec5725542a9
Top:    4b9c00872f04d4b6f88d4006f53a90777eafcc33
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:26:47 +0800

Refresh of autofs-5.0.7-fix-portmap-lookup.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 1f35b01..25f8968 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -358,7 +358,7 @@ static rpcprog_t rpc_getrpcbyname(const rpcprog_t program)
 	return program;
 }
 
-static unsigned short rpc_getservbyname(const char *service, const int protocol) 
+static unsigned short rpc_getservbyname(const char *service, const int protocol)
 {
 	const struct addrinfo hints = {
 		.ai_family      = AF_INET,
